pipelines.yml

```diff
@@ -86,6 +96,42 @@ pipelines:
           onComplete:
             - update_commit_status buildJfrogCliPluginGit --context "$step_name"
 
+  - name: upgrade_plugin_on_core_release
+    configuration:
+      runtime:
+        type: image
+        image:
+          custom:
+            name: releases-docker.jfrog.io/jfrog-ecosystem-integration-env
+            tag: 1.0.0
+    steps:
+      - name: upgrade_plugin_on_core_release
+        type: Bash
+        configuration:
+          inputResources:
+            - name: jfrog_cli_core_trigger
+          integrations:
+            - name: ILAutomation2
+            - name: JFrogCLIPluginsRegBot
+          onExecute:
+            - git clone https://github.com/jfrog/jfrog-cli-plugins.git && jfrogCliPluginsPath=`pwd`/jfrog-cli-plugins
+            - git clone https://github.com/jfrog/jfrog-cli-plugins-reg.git && cd jfrog-cli-plugins-reg/pipelinesScripts/validator
+            - env -i PATH=$PATH issue_token=$int_JFrogCLIPluginsRegBot_token HOME=$HOME PWD=$PWD JAVA_HOME=$JAVA_HOME GRADLE_HOME=$GRADLE_HOME MAVEN_HOME=$MAVEN_HOME go run validator.go upgradejfrogplugins $jfrogCliPluginsPath
+            - cd $jfrogCliPluginsPath
+            # Override origin.
+            - git remote set-url origin https://github.com/jfrog/jfrog-cli-plugins.git && git fetch origin main && git checkout main
+            # Check if new commits were added.
+            - "[[ $(git log origin/main...main --oneline | wc -l) -eq 0 ]]"
+            # Check if staged files contains none .mod or .sum files.
+            - "[[ $(git diff --name-only --staged | grep -v -e .mod$ -e.sum$ | wc -l) -eq 0 ]]"
+            # Check total number of staged files is not greater than twice the number of directories (plugins).
+            - "[[ $(git diff --name-only --staged | wc -l) -lt $(($(ls -l | grep ^d | wc -l)*2+1)) ]]"
+            # Commit and push
+            - git config user.email "orgab+2@jfrog.com"
+            - git config user.name "EcoSystem-bot"
+            - git commit -m "Upgrade jfrog plugins to jfrog-cli-core $res_jfrog_cli_core_trigger_releaseName"
+            - git push https://$int_ILAutomation2_token@github.com/jfrog/jfrog-cli-plugins.git main
+
```

Comment:
We should sent a notification email to eco-system-dev@jfrog.com in case there was a fatal error during the upgrade process. Otherwise, we'll never know about it.

Reply:
Since we don't have an SMTP server to send emails, we will open a GitHub issue on `jfrog-cli-plugins-reg` project.